app/assets/javascripts/backbone/views/project_step_view.coffee

```diff
@@ -91,5 +91,15 @@ class MS.Views.ProjectStepView extends Backbone.View
 
   deleteLog: (e, response) ->
     $.post @$(e.target).attr('href'), {_method: 'DELETE'}
+
+    # Number of logs currently visible includes one just deleted from database
+    logsList = @$(e.target).closest('.logs-list')
+    numOfLogs = @$(logsList).children().length
+
+    if numOfLogs < 2
+      logContainer = @$(logsList).closest('.step-logs')
+      @$(logContainer).removeClass('expanded')
+      @$(logContainer).find('.view-logs-ctrl').html('<span class="no-logs">No Logs</span>')
```

Comment:
Better to show/hide the existing than to insert one like this. The HTML you're inserting is not translated, for one. This is why it's best to keep HTML out of JS if at all possible.